src/main/java/com/slimebot/level/RankCard.java

```diff
@@ -0,0 +1,75 @@
+package com.slimebot.level;
+
+import com.slimebot.graphic.CustomFont;
+import com.slimebot.graphic.Graphic;
+import com.slimebot.graphic.ImageUtil;
+import com.slimebot.main.Main;
+import net.dv8tion.jda.api.entities.User;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.net.URL;
+
+public class RankCard extends Graphic {
+
+    private final static Font font;
+
+    static {
+        Font font_;
```

Comment:
Rename font_ to font and replace the last line with RankCard.font = font

Reply:
this is not possible because RankCard.font is final. if a variable is final it cannot be set via a class or via a class reference. with a static final variable it must be set directly in the class. (Without writing the class first. so no RankCard.font).

Reply:
Ok yeah, thats just java being weird, what I said should logically be possible because it des the same thing but is just referencing thing in a different way but java doesn't like it.